docs/developers/dapp-developer/native.md

```diff
@@ -86,17 +86,24 @@ nonce, err := cl.NonceAt(context.Background(), addr, big.NewInt(14000000))
 
 ### Querying past events {#querying-past-events}
 
-Contracts emit events during execution which can be queried from the client. The parameters for the event one is interested in have to be filled out in the `ethereum.FilterQuery` object. This includes which event topics are of interested, from which contracts and during which range of blocks. The example below queries `Transfer` events of all ERC-20 tokens for the last 10 blocks:
+Contracts emit events during execution which can be queried from the client. The parameters for the event one is interested in have to be filled out in the `ethereum.FilterQuery` object. This includes which event topics are of interest, from which contracts and during which range of blocks. The example below queries `Transfer` events of all ERC-20 tokens for the last 10 blocks:
 
 ```go
 blockNum, err := cl.BlockNumber(context.Background())
 if err != nil {
     return err
 }
+
+transferEventSignature := []byte("Transfer(address,address,uint256)")
+transferEventHash := crypto.Keccak256Hash(transferEventSignature)
+transferEventHashHexString = transferEventHash.Hex() // "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef"
```

Comment:
The way you changed it, it becomes a bit overly complicated. Why convert it to hex-string, just to convert it back further below?

```suggestion
// "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef"
transferEventHash := crypto.Keccak256Hash([]byte("Transfer(address,address,uint256)")) 
```